src/components/AboutNavButton/AboutNavButton.jsx

```diff
@@ -0,0 +1,14 @@
+import './AboutNavButton.css';
+import { Link } from 'react-router-dom';
+
+const AboutNavButton = () => {
+  return (
+    <Link className='nav-link' to='/about'>
+      <button className='nav-button font-face-modesto-expanded'>
```

Comment:
i support it, but why did you use the font face in the button class rule instead of the css? is there a trick I dont know about? please teach me your ways

Reply:
@Jesuitman That is actually the name of the class. I have the class styling defined in the `App.css` file so I can apply the font via class on any element across the app.